AppRater/src/main/java/org/codechimp/apprater/AppRater.java

```diff
@@ -113,13 +131,16 @@ public static void app_launched(Context context) {
      * @param context
      * @param daysUntilPrompt
      * @param launchesUntilPrompt
+     * @param significantEventUntilPrompt
      * @param daysForRemind
      * @param launchesForRemind
+     * @param significantEventForRemind
      */
-    public static void app_launched(Context context, int daysUntilPrompt, int launchesUntilPrompt, int daysForRemind, int launchesForRemind) {

```

Comment:
Agreed, it would break existing compatibility but with yours and other feature requests it makes sense.

Unfortunately I don't have time to do this immediately and so will not only be maintaining current features for now but will start a new branch for a V2 in future.